src/_forms.scss

```diff
@@ -59,7 +59,7 @@ input, textarea {
 	&::placeholder {
```

Comment:
Can we convert this file to be indented by spaces?

Reply:
I can do it, but it doesn't register as change :/